src/components/value-list/value-list.tsx

```diff
@@ -38,8 +38,8 @@ import { createObserver } from "../../utils/observers";
 import { InteractiveComponent, updateHostInteraction } from "../../utils/interactive";
 
 /**
- * @slot - A slot for adding `calcite-value-list-item` elements. Items are displayed as a vertical list.
- * @slot menu-actions - A slot for adding a button + menu combo for performing actions like sorting.
+ * @slot - A slot that adds calcite-value-list-item elements. List items are displayed as a vertical list.
```

Comment:
Same here. A slot is a placeholder for adding elements so I think `for adding` actually makes more sense than `that  adds`. What do you think?